builtin/game/item.lua

```diff
@@ -1,4 +1,22 @@
--- Minetest: builtin/item.lua
+---------------------------------------------------------------------------------
```

Comment:
None of the files in builtin/ have a copyright header. And if you add one, can you at least watch out that the year matches the current one?